<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_welcome</class>
 <widget class="QDialog" name="Dialog_welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <widget class="QLabel" name="label_1_welcome">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ウェルカム</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>このウィザードに従って、VRFシミュレーターのスキャンデータを作成することができます。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>作成作業を行う場合、“次へ”ボタンを、キャンセルの場合“キャンセル”ボタンを押下してください。</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_cancel_welcome">
      <property name="text">
       <string>キャンセル</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_next_welcome">
      <property name="text">
       <string>次へ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_pic_welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
